app/common/locales/en.json

```diff
@@ -1523,8 +1523,10 @@
             "save_attribute_success" : "Field {{name}} added",
             "edit_form_success" : "Survey {{name}} updated",
             "edit_stage_success" : "Survey task {{name}} updated",
-            "delete_form_confirm" : "Are you sure you want to delete this survey?",
-            "delete_form_confirm_desc" : "This action cannot be undone. Deleting this survey will remove all of its data, including posts.",
+            "delete_form_confirm" : "Are you sure you want to delete this survey and all its posts?",
+            "delete_form_confirm_desc" : "This action cannot be undone. Deleting this survey will remove all of its data, including posts. Write the name of the survey: <br><br><strong>{{check_name}}</strong><br><br>in the box below to confirm deletion.",
+            "delete_form_error": "The text you entered does not match the survey-name, please try again",
+            "delete_form_button": "Delete this survey",
```

Comment:
`Delete this survey and all its data` maybe? or did we go against that in our chat? can't recall

Reply:
I don't think we talked about it. Do you think people understand that this means posts?